src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -166,6 +166,8 @@ private void parseQueryMeta(ResultSet rsQueryMeta) throws SQLServerException {
                             qm.precision = 8;
                         } else if (SSType.XML == ssType) {
                             qm.precision = SQLServerDatabaseMetaData.MAXLOBSIZE / 2;
+                        }  else if (SSType.JSON == ssType) {
+                            qm.precision = SQLServerDatabaseMetaData.MAXLOBSIZE / 2;
```

Comment:
What is the meaning of `precision` in JSON's context ?

Reply:
Actually, we followed the XML datatype code path, so added it, but it is not required. Tested the code.